index.bs

```diff
@@ -0,0 +1,177 @@
+proximity sensor. The proximity level is reported as the distance (in centimeter)
+from the sensor to the closest visible surface. The precise distance value
+reported by different devices can be different, due to differences in detection
+method, sensor construction, etc. Moreover, some proximity sensors might be only
+able to provide just a boolean to indicate if there is an object which is near,
+more like presence detection, than an absolute value for the distance.
+
+Since most proximity sensors detect electromagnetic radiation (e.g., an infrared light
+or a magnetic field), certain material properties can interfere with the sensor's
+ability to sense the presence of a physical object. Things that can interfere with
+a sensor include, but are not limited to, the material's translucency, reflectiveness,
+color, temperature, chemical composition, and even the angle at which the object
+is reflecting the radiation back at the sensor. As such, proximity sensors
+should not be relied on as a means to measure distance. The only thing that can
+be deduced from a proximity sensor is that an object is somewhere in the
+distance between the minimum sensing distance and the maximum sensing distance
+with some degree of certainty.
+
+
+Examples {#examples}
+========
+
+<div class="example">
+    <pre highlight="js">
+    let sensor = new ProximitySensor();
+    sensor.start();
+
+    sensor.onchange = event => console.log(event.reading.distance);
+
+    sensor.onerror = function(event) {
+        console.log(event.error.name, event.error.message);
+    };
+    </pre>
+</div>
+
+Security and Privacy Considerations {#security-and-privacy}
+===================================
+
+There are no specific security and privacy considerations
+beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
+
+Model {#model}
+=====
+
+The Proximity Sensor's associated <a>Sensor subclass</a>
+is the {{ProximitySensor}} class.
+
+The Proximity Sensor's associated <a>SensorReading subclass</a>
+is the {{ProximitySensorReading}} class.
+
+The Proximity Sensor has a <a>default sensor</a>,
+which is the device's main proximity detector.
+
+The Proximity Sensor has a single <a>supported reporting mode</a>
+which is "<a>auto</a>".
+
+The Proximity Sensor's <a>permission</a> name is `"proximity"`.
+It has no <a>associated PermissionDescriptor</a>.
+
+The Proximity Sensor has an associated abstract operation
+to <dfn>retrieve the sensor permission</dfn> which
+must simply return a <a>permission</a> whose name is "proximity".
+
+The Proximity Sensor has an associated abstract operation
+to <dfn lt="Construct SensorReading Object">construct a SensorReading object</dfn>
+which creates a new {{ProximitySensorReading}} object and sets its
+<a attribute for="ProximitySensorReading">distance</a> attribute to
+positive infinity.
+
+<dfn>Distance</dfn> is a value that represents the distance between a device and
+the closest visible surface. Its unit is the centimetre (cm).
```

Comment:
cm was already the unit we used in the previous proximity spec; see the [thread with the reasoning behind that choice](https://lists.w3.org/Archives/Public/public-device-apis/2012May/0196.html)

Not all proximity sensors provide it, and we will need confirm the value for sensors that don't (null?), but some do, and it would be a shame not to expose it.